hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/serialization/Expirable.java

```diff
@@ -94,4 +95,20 @@ public void readData(final ObjectDataInput in) throws IOException {
         version = in.readObject();
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Expirable expirable = (Expirable) o;
+        return Objects.equals(version, expirable.version);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(version);
```

Comment:
Could it be
```suggestion
        return Objects.hashCode(version);
```
to eliminate parameter list array?